subprojects/dependency-management/src/integTest/groovy/org/gradle/integtests/resolve/suppliers/DynamicRevisionRemoteResolveWithMetadataSupplierIntegrationTest.groovy

```diff
@@ -819,6 +819,9 @@ group:projectB:2.2;release
         }
         supplierInteractions.refresh('group:projectB:2.2', 'group:projectB:1.1')
         checkResolve "group:projectA:1.+": ["group:projectA:1.2", "didn't match version 2.0"], "group:projectB:latest.release": ["group:projectB:1.1", "didn't match version 2.2"]
+
+        where:
```

Comment:
I would not change tests in that class, as they are conceptually not focused on testing dependency refresh.
Tests in `ResolutionOverrideIntegrationTest` are better candidates.

Reply:
Got it, moved